pom.xml

```diff
@@ -57,6 +57,10 @@
   </scm>
 
   <repositories>
+    <repository>
+      <id>apache</id>
+      <url>https://repository.apache.org/content/repositories/public/</url>
```

Comment:
Which repository was it getting the dependencies from previously? (where is that defined)